deploy-manage/toc.yml

```diff
@@ -456,16 +456,21 @@ toc:
           - file: security/secure-your-eck-installation.md
       - file: security/secure-your-cluster-deployment.md
         children:
+          - file: security/security-certificates-keys.md
+          - file: security/set-up-minimal-security.md
           - file: security/secure-cluster-communications.md
             children:
-              - file: security/secure-http-communications.md
-              - file: security/k8s-transport-settings.md
-              - file: security/security-certificates-keys.md
+              - file: security/set-up-basic-security.md
+              - file: security/set-up-basic-security-plus-https.md
+              - file: security/updating-certificates.md
                 children:
-                  - file: security/updating-certificates.md
-                    children:
-                      - file: security/same-ca.md
-                      - file: security/different-ca.md
+                  - file: security/same-ca.md
+                  - file: security/different-ca.md
+              - file: security/kibana-es-mutual-tls.md
+              - file: security/eck-tls.md
+                children:
+                  - file: security/k8s-https-settings.md
+                  - file: security/k8s-transport-settings.md
               - file: security/supported-ssltls-versions-by-jdk-version.md
               - file: security/enabling-cipher-suites-for-stronger-encryption.md
```

Comment:
might want to intro these two pages in the parent page

Reply:
Yes, I'll find the way, thanks!